examples/collector/collector.cpp

```diff
@@ -50,16 +50,17 @@ std::ostream &operator<<(std::ostream &os,
 
 /**
  * A map of functions that format the parameters and arguments for each UR function.
- * This example only implements a handler for one function, `urInit`, but it's
```

Comment:
this patch makes it impossible to trace this function, so this example doesn't work with `urLoaderInit`. It should probably be changed to something else that is traceable.

Reply:
As part of the recent loader changes, there's now one additional env variable to set `UR_ENABLE_LAYERS=UR_LAYER_TRACING`. README needs updating.

This runs in CI, and I've noticed because you had to change the [hello_world output](https://github.com/oneapi-src/unified-runtime/pull/793/files#diff-beee61f5a9e76bfc5266df9381f10343c24bda0775f9d612fc0c2e29d58619d4) ;)

Reply:
I didn't even notice that hello_world.output belonged to this test, it's fixed now! README is also updated.